channels.json

```diff
@@ -61,8 +61,8 @@
       "url": "https://namtx.dev/feed.xml"
     },
     {
-      "name": "Coder7een",
-      "avatar_url": "",
+      "name": "coder seventeen",
+      "avatar_url": "https://raw.githubusercontent.com/coder7eeN/coder7eeN.github.io/main/images/avatar_rss.png",
```

Comment:
Sorry but we do not support loading avatars from an URL yet, could you pls add this file to public/avatars folder?

Reply:
yes, I will do that

Reply:
ok done, pls check it again, thank you